app/src/main/java/projekt/substratum/common/commands/FileOperations.java

```diff
@@ -328,6 +334,9 @@ public static void move(Context context, String source, String destination) {
             } catch (InterruptedException e) {
                 Thread.interrupted();
             }
+            if (!file.exists()) {
+                move(source, destination);
```

Comment:
So you are checking if things aren't moving by ThemeInterfacer, you will ask the app too? 

This should be impossible to reach as we use the ThemeInterfacer to move files that third party apps cannot move, such as /data/system/theme folder.